app/src/main/java/io/apicurio/registry/util/SearchUtil.java

```diff
@@ -73,6 +73,7 @@ public static SearchedVersion buildSearchedVersion(ArtifactVersionMetaDataDto ar
         searchedVersion.setState(artifactVersionMetaData.getState());
         searchedVersion.setType(artifactVersionMetaData.getType());
         searchedVersion.setVersion(artifactVersionMetaData.getVersion());
+        searchedVersion.setLabels(artifactVersionMetaData.getLabels());
 
```

Comment:
No properties here?

Reply:
I'm leaving out properties from the search results for now.  We can discuss if they should be included or not.